handlers/validator.go

```diff
@@ -359,58 +360,60 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 			break
 		}
 	}
+	// if we are currently past the cappella fork epoch, we can calculate the withdrawal information
+	if epoch >= utils.Config.Chain.Config.CappellaForkEpoch {
+		// get validator withdrawals
+		withdrawalsCount, err := db.GetValidatorWithdrawalsCount(validatorPageData.Index)
+		if err != nil {
+			logger.Errorf("error getting validator withdrawals count from db: %v", err)
+			http.Error(w, "Internal server error", http.StatusInternalServerError)
+			return
+		}
+		validatorPageData.WithdrawalCount = withdrawalsCount
 
-	// get validator withdrawals
-	withdrawalsCount, err := db.GetValidatorWithdrawalsCount(validatorPageData.Index)
-	if err != nil {
-		logger.Errorf("error getting validator withdrawals count from db: %v", err)
-		http.Error(w, "Internal server error", http.StatusInternalServerError)
-		return
-	}
-	validatorPageData.WithdrawalCount = withdrawalsCount
-
-	validatorPageData.ShowWithdrawalWarning = hasMultipleWithdrawalCredentials(validatorPageData.Deposits)
```

Comment:
At least the `ShowWithdrawalWarning` information has to be shown even pre-Capella. Please move it outside of this if.

`IsWithdrawableAddress` on the other hand is a bit more tricky. It should be set to true if the validator has 0x01 creds even pre-Capella, right? This would hide the following warning:
![image](https://user-images.githubusercontent.com/114066135/220880995-4afaa1bc-7a0c-4618-9ffa-e603f98c6e10.png)
On the other hand, the Withdrawals tab would become active even pre-Capella which would not be perfect.

Thoughts?

**Sidenote:** Now that withdrawals are supported by the explorer, the variable name `ShowWithdrawalWarning` is too vague and I suggest to change it to something like `ShowMultipleWithdrawalCredentialsWarning`.

Reply:
Ah yes I think I went a bit overboard with this, should move this outside the capella check

Reply:
All the suggestions sound good